wbooks/src/app/components/FlatListWrapper/index.tsx

```diff
@@ -0,0 +1,43 @@
+// Taked from: https://github.com/facebook/react-native/issues/31697#issuecomment-860447695
+// Solve "VirtualizedLists should never be nested inside plain ScrollViews with the same orientation"
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import React from 'react';
+import { FlatListProps, FlatList as NativeFlatList, View } from 'react-native';
+
+// Custom FlatList that doesnt try to virtualize when scrollEnabled={false}
+// Do this to avoid "VirtualizedLists should never be nested inside plain ScrollViews with the same orientation" error in console
+export default function FlatListWrapper<ItemT>(props: FlatListProps<ItemT>) {
```

Comment:
what is the utility of this component?

Reply:
I take it from [here](https://github.com/facebook/react-native/issues/31697#issuecomment-860447695), the purpose it's remove the warning "VirtualizedLists should never be nested inside plain ScrollViews with the same orientation" when use a ScrollView and a FlatList. Without the ScrollView the view isnt be scrolleable.

Reply:
Should i remove it? Exists another solution?👀

Reply:
there are other solutions, but this is ok for the training